test/python/test_tools_database_import.py

```diff
@@ -200,3 +200,26 @@ def test_load_data(dsn, src_dir, place_row, placex_table, osmline_table, word_ta
 
     assert temp_db_cursor.table_rows('placex') == 30
     assert temp_db_cursor.table_rows('location_property_osmline') == 1
+
+@pytest.mark.parametrize("languages", (False, True))
+def test_create_country_names(temp_db_conn, temp_db_cursor, def_config,
+                              temp_db_with_extensions, monkeypatch, languages):
+    if languages:
+        monkeypatch.setenv('NOMINATIM_LANGUAGES', 'fr,en')
+    temp_db_cursor.execute("""CREATE FUNCTION make_standard_name (name TEXT)
+                                  RETURNS TEXT AS $$ SELECT 'a'::TEXT $$ LANGUAGE SQL
+                               """)
+    temp_db_cursor.execute("""CREATE OR REPLACE FUNCTION getorcreate_country(lookup_word TEXT,
+                                               lookup_country_code varchar(2))
+                            RETURNS INTEGER
+                            AS $$
+                            BEGIN
+                                INSERT INTO country_name VALUES (5, lookup_word);
+                                RETURN 5;
+                            END;
+                            $$
+                            LANGUAGE plpgsql;
+                               """)
```

Comment:
You can't fake the function like that. The purpose of create_country_names() is to take information from the country_name table and add it to the word table. If you insert values into country_name in your test getorcreate_country() you invert the purpose.

Reply:
So Should I recreate the whole function?

Reply:
No, a simple function like this is fine. Just create an extra table to insert the lines to, so that you do not change the input table `country_name`.

Reply:
Sure will do this then. Actually, initially, my idea was this only. But later, I thought that it would be better to use a country table instead of creating a new table so that I don't have to create a new table.